source/threads.tex

```diff
@@ -6432,7 +6385,6 @@
 
 using std::@\libglobal{memory_order}@;             // \seebelow
 using std::@\libglobal{memory_order_relaxed}@;     // \seebelow
-using std::@\libglobal{memory_order_consume}@;     // \seebelow
```

Comment:
Should this be moved to Annex D rather than removed outright?

Reply:
Maybe.  The paper contains no mention of this header; I have no idea what the LEWG-approved design intent here might be.

Reply:
@jwakely, what do you think?

Reply:
Removed that change by force-pushing.